BitFaster.Caching/Lfu/CmSketchCore.cs

```diff
@@ -120,8 +125,20 @@ private void EnsureCapacity(long maximumSize)
         {
             int maximum = (int)Math.Min(maximumSize, int.MaxValue >> 1);
 
+#if NET6_0_OR_GREATER
+            // over alloc by 4 to give 32 bytes padding, tableAddr is then aligned to 32 bytes
+            const int pad = 4;
+            table = GC.AllocateArray<long>(Math.Max(BitOps.CeilingPowerOfTwo(maximum), 8) + pad, true);
```

Comment:
We can choose to pin here only if AVX is enabled, else it can be a normal array.